docs/user-guides/dashboard-as-code.md

```diff
@@ -66,6 +66,108 @@ If the build is successful, the result can be found in the generated `built` fol
 > [!NOTE]
 > the `-o` (alternatively '--output') flag is optional (the default output format is YAML).
 
+## Getting started with Go SDK
+
+### Prerequisites
+
+- `percli`, the [CLI of Perses](../tooling/cli.md).
+- `go`, the [programming language](https://go.dev/).
+
+### Repository setup
+
+Create a new folder that will become your DaC repository, then follow the steps below:
+
+#### 1. Initialize the Go module
+
+```
+go mod init <module name>
```

Comment:
that's something that the command `percli dac setup` could handle.

Reply:
there's the same step on the CUE side (cue.mod), imho in that case we should include mod file generation either in none or both.